clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/agent/KubernetesCacheDataConverter.java

```diff
@@ -127,15 +123,12 @@ public static void convertAsResource(
     kubernetesCacheData.addItem(key, attributes);
 
     SpinnakerKind spinnakerKind = kindMap.translateKubernetesKind(kind);
-    if (logicalRelationshipKinds.contains(spinnakerKind)
-        && !Strings.isNullOrEmpty(moniker.getApp())) {
-      addLogicalRelationships(
-          kubernetesCacheData,
-          key,
-          account,
-          moniker,
-          clusterRelationshipKinds.contains(spinnakerKind));
-    }
+    addLogicalRelationships(
```

Comment:
This change adds application relationships for every resource. Not only the ones in the `logicalRelationshipKinds` list